### PR TITLE
[Tools] Removed query statements causing installdb.php to fail

### PR DESCRIPTION
**Pull request by AnyhowStep**
Running `installdb.php` when installing Loris fails. The fix is to remove the `SELECT`, `WARNINGS;` and `SET SQL_NOTES=1;` statements. Doing so allows `installdb.php` to function as desired.

Those statements cause `installdb.php` to fail because the script uses `PDO::exec()`, via `Installer.class.inc`, internally. `PDO::exec()` cannot return results and cannot clear results from its buffer, as far as I know. The subsequent call to `PDO::exec()` fails because the previous call returned results and are still stuck in the buffer. The error message returned is related to the buffer still having results.

Possible alternative fix would be to reset the database connection after each PDO::exec() call.

### CODE REVIEW

File: SQL/0000-00-03-ConfigTables.sql [hunk 1]
```diff
@@ -1,19 +1,18 @@
 WARNINGS;
```
Comment (xlecours) [inline SQL/0000-00-03-ConfigTables.sql]: Should be removed as well

**Comment by xlecours** [comment]
Tested with first line of SQL/0000-00-03-ConfigTables.sql  removed

File: SQL/0000-00-03-ConfigTables.sql (hunk 1 above)
Comment (AnyhowStep) [inline SQL/0000-00-03-ConfigTables.sql]: Just removed it

**Comment by AnyhowStep** [comment]
Just made the change to `SQL/0000-00-03-ConfigTables.sql`

**Comment by ridz1208** [comment]
@xlecours please re-review

**Pull request merged**